library/core/src/intrinsics/simd.rs

```diff
@@ -754,6 +812,7 @@ pub unsafe fn simd_fsqrt<T>(x: T) -> T;
 /// `T` must be a vector of floats.
 #[rustc_intrinsic]
 #[rustc_nounwind]
+#[rustc_intrinsic_const_stable_indirect]
 pub const unsafe fn simd_fma<T>(x: T, y: T, z: T) -> T;
```

Comment:
For this, not even the scalar version is const-stable yet. It seems odd to stabilize the SIMD version before the scalar version.